### ISSUE TITLE
No bars on colored, faceted chart, sorted by aggregate value

### ISSUE THREAD

**Issue by emonson** [hex-inc/vegafusion#391]
*(Thanks for all of the great work – I find VegaFusion super useful!!)*

I get a blank mark_bar chart (in JupyterLab) if I do a combination of faceting and coloring by a category, plus sorting the shared axis by an aggregated numeric value. The chart looks good with the default Altair renderer, and it displays fine with VF if I take away the sorting, the coloring, or the faceting, or if I switch to mark_point, circle, line, or rect.

```
import pandas as pd
import altair as alt
import vegafusion as vf
vf.enable()

df = pd.DataFrame({'location':['US','US','US','UK','UK','UK'],
                    'category':['low','high','high','low','high','high'],
                    'value':[1,2,3,2,3,4]})

alt.Chart(df).mark_bar().encode(
    x = 'sum(value):Q',
    y = alt.Y('category:N').sort(alt.EncodingSortField(field='value',op='sum',order='descending')),
    color = 'location:N'
).facet(
    column = 'location:N'
)
```

![visualization (2)](https://github.com/hex-inc/vegafusion/assets/294412/eb31738b-aa96-4a2f-88ae-4edfcee941bf)

Versions: 
- vegafusion & vegafusion-jupyter 1.4.0
- altair 5.0.1
- python 3.11.5 (anaconda dist)
- jupyterlab 3.6.3
- mac os 12.6.9 / apple M1 pro / chrome

**Comment by jonmmease** [comment]
Thanks for the report!

**Comment by jonmmease** [comment]
This will be fixed in the next release (vegafusion 1.5.0). Please keep the reports coming as you run into things, we want VegaFusion to be something you (and your students) can turn on without worrying about compatibility or caveats.

Your course looks great BTW!

**Comment by emonson** [comment]
Awesome! Thanks a lot!
